source/extensions/filters/network/common/redis/codec_impl.cc

```diff
@@ -458,6 +595,20 @@ void EncoderImpl::encodeArray(const std::vector<RespValue>& array, Buffer::Insta
   }
 }
 
+void EncoderImpl::encodeCompositeArray(const RespValue::CompositeArray& composite_array,
+                                       Buffer::Instance& out) {
+  char buffer[32];
+  char* current = buffer;
+  *current++ = '*';
+  current += StringUtil::itoa(current, 31, composite_array.size());
```

Comment:
nit: I would use 20 here which I think is the actual max size.

Reply:
I'll change it, it should be 21.